modules/components/core/org.wso2.carbon.pc.core/src/main/java/org/wso2/carbon/pc/core/ProcessStore.java

```diff
@@ -764,28 +764,32 @@ public String getSucessorPredecessorSubprocessList(String resourcePath){
                                 subprocessElement.getElementsByTagName("path").item(0).getTextContent();
                         String subprocessId =
                                 subprocessElement.getElementsByTagName("id").item(0).getTextContent();
+                        String subprocessVersion = subprocessPath.substring(subprocessPath.lastIndexOf("/") + 1).trim();
 
                         JSONObject subprocess = new JSONObject();
                         subprocess.put("name", subprocessName);
                         subprocess.put("path", subprocessPath);
                         subprocess.put("id", subprocessId);
+                        subprocess.put("version", subprocessVersion);
                         subprocessArray.put(subprocess);
                     }
                 }
 
                 if(successorElements.getLength() != 0){
```

Comment:
WSO2 coding style needs to be applied here. "if" needs to be followed with a space before the curly bracket. This needs to be fixed in several places.